src/main/java/org/vanilladb/core/storage/record/RecordFile.java

```diff
@@ -107,6 +107,14 @@ public void close() {
 	}
 
 	/**
+	 * Remove the record file.
+	 */
+	public void remove() {
```

Comment:
There might be some concurrency issues here. For example, if a transaction is modifying a table, and another transaction is dropping that table, unpredictable results may happen, which may violate SERIALIZABLE isolation.
However, I think it is almost impossible to occur such scenarios in the real world, since we usually drop a table when we know that on one is using it. Therefore, you can just leave a comment `TODO: to handle the concurrency issues might happen here` here for future implementation.